src/sas/system/log.py

```diff
@@ -39,6 +40,7 @@ def config_development(self):
         self._update_all_logs_to_debug(logger)
         logging.captureWarnings(True)
         logging.getLogger('matplotlib').setLevel(logging.WARN)
+        logging.getLogger('numba').setLevel(logging.WARN)
```

Comment:
Changing level to WARNING for handler_log_file and logger_root in log.ini seems like a better solution.

Reply:
For releases it makes sense to change default logging level to WARNING.
For developer versions it would probably be better to leave it on DEBUG.
This means adding complexity to the logger logic and should be addressed in the config options PR or in a separate issue.

Reply:
Line 40: (_update_all_logs_to_debug) changes all our libraries to DEBUG so that we don't have to set it as the default.

For third party libraries we should whitelist any that we want to debug not blacklist any that make too much noise.

Reply:
Agreed.
Will be addressed in https://github.com/SasView/sasview/issues/2289